client/src/components/History/CurrentHistory/HistoryOperations/SelectionOperations.vue

```diff
@@ -60,6 +74,57 @@
             <p v-localize>Permanently delete {{ numSelected }} content items?</p>
             <p><strong v-localize class="text-danger">Warning, this operation cannot be undone.</strong></p>
         </b-modal>
+        <b-modal
+            id="change-dbkey-of-selected-content"
+            title="Change reference gnome?"
+            title-tag="h2"
+            @ok="changeDbkeyOfSelected">
+            <p v-localize>Select a new reference genome for {{ numSelected }} items:</p>
+            <GenomeProvider v-slot="{ item: genomes, loading: loadingGenomes }">
```

Comment:
```suggestion
            <DbKeyProvider v-slot="{ item: genomes, loading: loadingGenomes }">
```
and all the changes that follow from this ...

Reply:
I can't find an existing `DbKeyProvider`. Do you mean renaming the existing `GenomeProvider` to `DbkeyProvider` and all the existing references?

Reply:
Yes, it shouldn't have been named `GenomeProvider`.

Reply:
Ohh I see, I will rename the provider and all its references in a follow-up PR, just created #13964 so I don't forget to do so :+1: